src/index.html

```diff
@@ -4,13 +4,65 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, userScalable=no, initialScale=1.0, maximumScale=1.0, minimumScale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+      class="form"
+    >
+      <label
+        class="form-label top-search-bar" for="bigSearch"
```

Comment:
tiny one - attribute to a new line
```suggestion
        class="form-label top-search-bar"
        for="bigSearch"
```